Style dat details header

Kriesse opened the pull request:
Styles for dat-header.

Clean up main.css:
- Remove unused styles
- Rename unclear class names
- Namespace global styles
- Used classes, not IDs when possible
- Group chunks of scss in separate files. Moving forward, this should make it easier to move styles that we want to reuse elsewhere over to dat-design.

karissa commented:
Looks awesome! Great job

karissa commented:
Saw this happen with the speed widget on breakpoint sm
<img width="665" alt="screen shot 2016-07-21 at 11 56 58 am" src="https://cloud.githubusercontent.com/assets/633012/17035089/706b2be6-4f3a-11e6-81b1-d0b0f267c3c9.png">

Kriesse commented:
@karissa Cool, thanks for reviewing! Added some breakpoint tweaks, the layout should work now down to a width of 480px (I'd like to spend some time on optimizing it for even smaller screens, but later). 

I also added placeholders for the "Download" and "Open in Desktop" buttons.

clkao commented:
oops, there are a conflicts with the e2e tests changes (mostly adding the upload button) i just merged

karissa commented:
how is this now? I see the travis build didn't pass on push regarding the # of sources, but I'm not sure that's related to this PR (seems like it could be an issue with hyperdrive 7)

clkao commented:
triggering a rebuild might help. travis is broken for me right now though

Kriesse commented:
@karissa @clkao Thanks, rebuilding seemed to have helped!
